packages/0x.js/test/expiration_watcher_test.ts

```diff
@@ -12,16 +13,16 @@ import {DoneCallback, Token} from '../src/types';
 import {constants} from '../src/utils/constants';
 import {utils} from '../src/utils/utils';
 
-import {BlockchainLifecycle} from './utils/blockchain_lifecycle';
 import {chaiSetup} from './utils/chai_setup';
+import {constants as testConstants} from './utils/constants';
```

Comment:
Everywhere else this is just imported as `constants`. Don't mind either way, just remain consistent.

Reply:
Because no other file imports both constants files. The one from src and the one from tests. So I have a collision.